### ISSUE TITLE
[FEATURE REQUEST] ETA until the next filament swap

### ISSUE THREAD

**Issue by Ferferite** [prusa3d/Prusa-Firmware-Buddy#1681]
**Printer type** - [MINI/MINI+]
**Optional upgrades** - none

**Is your feature request related to an existing problem? Please describe.**
  Let's say you are waiting on a big print, but you have a filament swap somewhere in the print. Currently, you don't have any easy way to know *when* that filament swap will happen. So it would be nice to have a timer on the screen that tells you in how much time the printer will pause to do the filament swap. 

**Describe the expected functionality**
 It could have a small timer next to the progress bar, or maybe the progress bar could be in 2 different colors, kinda like the slicer.

**Comment by Gyrxiur** [comment]
Nice idea, right now I am dealing with same issue.
